server/src/index.ts

```diff
@@ -472,22 +471,15 @@ app.get('*', (_, res) => {
     path.join(__dirname, '..', '..', 'client', 'build', 'index.html')
   );
 });
-
 const debug = createDebug(`webhooks`);
 
-app.post('/webhooks', async (req, res) => {
-  const isWebhookVerified = await verifyWebhookSecret(req);
-  if (!isWebhookVerified) {
-    res.status(400).send('Request not verified');
-    debug(`Webhook verification failed for req ${req},`);
-  } else {
-    res.status(200).end();
-    githubPushWebhookHandler(isWebhookVerified);
+app.post('/api/webhooks', async (req, res) => {
+  debug('received request -----------------------------', req.body);
+
+  if (req.header('x-github-event') === 'push') {
+    await handleWebhooks(req);
   }
```

Comment:
shouldn't we return 200 at the end? Also we should use a try catch and return a 500 error in case handleWebhooks return an issue.